go/noise/noise.go

```diff
@@ -57,6 +59,22 @@ func ToKind(n Noise) Kind {
 	return GaussianNoise
 }
 
+// ConfidenceIntervalInt64 holds upper and lower bounds in int64 for the confidence interval.
```

Comment:
Please change the order of upper and lower in the documentation.

Also, "as int64" instead of "in int64".

(Same comment also below)